### DISCUSSION TITLE
[Script request] Vikunja

### DISCUSSION THREAD

**Discussion by CrazyWolf13** [community-scripts/ProxmoxVE#35] [Request script] [answered]
### Application Name

Vikunja

### Website

https://vikunja.io/

### Description

Plan your projects with the elegance
of a sloth on a sunny day.
Vikunja, the fluffy, open-source, [self-hostable](https://vikunja.io/docs/installing/) to-do app.

Made and hosted in the EU.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.

**Comment by wisie** [comment]
Another vote for Vikunja. Have been hoping to see this added for a while now.

**Comment by MickLesk** [comment, marked as answer]
Done.

```bash
bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/vikunja.sh)"

**Reply by wisie** [reply]
Amazing, thank you. Will spin it up in a bit.

**Reply by CrazyWolf13** [reply]
@MickLesk Thanks for creating this!

Just looked at it, and seems like there is no mysql db, afaik. vikunja always also needs a DB, here an example with their docker compose:
https://vikunja.io/docs/full-docker-example

**Reply by MickLesk** [reply]
Nope. Its a full dpkg Debian File.
There is an self DB (i think sqlite)

**Reply by CrazyWolf13** [reply]
ohhh I see, thanks for clarification!

**Comment by judeibe** [comment]
Closing as completed

**Reply by knario** [reply]
Hello, happy week to you all!
I have a question about vikunja and if anyone of you use it with android and davx5 and how did you get it. 
Because as much as I try I can't get it.
Thank you very much.

**Reply by CrazyWolf13** [reply]
@knario I suggest asking this directly to the project maintainer either via issue on their repo or on their matrix channel.

We only provide a install-script.

**Reply by knario** [reply]
Muchas gracias, ayer conseguí solucionarlo. Con sólo poner https://tu dominio/dav funcionó. 
Feliz semana.
